charts/retool/charts/retool-temporal-services-helm/templates/server-service.yaml

```diff
@@ -87,9 +87,9 @@ spec:
       protocol: TCP
       name: grpc-membership
     - port: 9090
-      targetPort: metrics
+      targetPort: http-metrics
       protocol: TCP
-      name: metrics
+      name: http-metrics
```

Comment:
would change this if also change above comment

Reply:
does changing it complicate anything? I was wondering the same, but OTOH I don't have the change if only for consistency of naming scheme, if it doesn't hurt anything.

Reply:
maybe a very minor downtime if service port changes before new pod comes up?
but yeah agree if we make them consistent we should do for all (e.g. the two ports defined above)

Reply:
oh right, we're using a named targetPort here. good point.

Reply:
actually I think technically, if this causes minor downtime, it's downtime either way, since we _have_ to update `targetPort` to match the pod's new port name which comes from the Deployment template. we could choose to or not to update `name` here on the Service object to match purely for consistency, but I don't think that would have any downtime since the Service port number (`port` here) doesn't change. I think?

Reply:
it seems like the ports that only need naming are on service and pod 

https://istio.io/v1.0/docs/setup/kubernetes/spec-requirements/

I think will stick with the current changes then 👍